exchange/exchange_test.go

```diff
@@ -7,6 +7,15 @@ import (
 	"github.com/valinurovam/garagemq/binding"
 )
 
+func contains(queues []string, queue string) bool {
```

Comment:
The `exists` function above does exactly the same as this, maybe both could be factorized in some place?